ant/src/process.lua

```diff
@@ -0,0 +1,175 @@
+-- Adjust package.path to include the current directory
+-- utils
+local json = require(".json")
+local utils = require(".utils")
+local errors = require(".errors")
+local constants = require(".constants")
+
+-- core modules
+local balances = require(".balances")
+local initialize = require(".initialize")
+local records = require(".records")
+local controllers = require(".controllers")
+
+Name = Name or "Arweave Name Token"
+Ticker = Ticker or "ANT"
+Logo = Logo or "LOGO"
+Denomination = Denomination or 1
+
+local ActionMap = {
+	-- write
+	SetController = "SetController",
+	RemoveController = "RemoveController",
+	SetRecord = "SetRecord",
+	RemoveRecord = "RemoveRecord",
+	SetName = "SetName",
+	SetTicker = "SetTicker",
+	--- initialization method for bootstrapping the contract from other platforms ---
+	InitializeState = "InitializeState",
+	-- read
+	GetControllers = "GetControllers",
+	GetRecord = "GetRecord",
+	GetRecords = "GetRecords",
+}
+
+local TokenSpecActionMap = {
```

Comment:
The token.lua blueprint handlers use lower case.  We should use the same.

https://github.com/permaweb/aos/blob/main/blueprints/token.lua

Reply:
Gotchya, specifically camel case.

The action tags use upper kebab case (mutant combo of pascal and kebab case) - added a util to convert all registered handlers to camel case.